content/soda-gettingstarted/installation-using-ansible.md

```diff
@@ -339,10 +331,13 @@ delfin_exporter_alertmanager_port: 9093
 --- 
 
 ### Configure SRM Toolchain installation
-
+Delfin produces metrics which can be consumed by any of the exporters that are supported. Currently Delfin supports the Prometheus and Kafka exporters.
+The SRM Toolchain is required to view the metrics and visualization in the SODA Dashboard.\  
 Installing the SRM toolchain will install Prometheus, AlertManager and Grafana versions as per the configuration below and can be changed. 
 
```

Comment:
These instructions are missing which file needs to be modified. 
This was available earlier. Why is it removed?
_Update the file `ansible/group_vars/srm-toolchain.yml` and change the value of `install_srm_toolchain` to `true`.  
If this value is set to false then the metrics and visualization will not be available using SODA Dashboard._

Reply:
Previously two blocks were there for SRM toolchain 
1st for definition and 2nd for configuration 
now we refactored and combined in both in one block 
and install_srm_toolchain is available and in the Delfin installation process install_srm_toolchain to true is given

Reply:
> Previously two blocks were there for SRM toolchain 1st for definition and 2nd for configuration now we refactored and combined in both in one block and install_srm_toolchain is available and in the Delfin installation process install_srm_toolchain to true is given

ok